owid/catalog/datasets.py

```diff
@@ -20,6 +20,9 @@
 from .meta import DatasetMeta, TableMeta
 from . import utils
 
+ALLOWED_FORMATS = ["csv", "feather", "parquet"]
+DEFAULT_FORMATS = ["csv", "feather", "parquet"]
```

Comment:
We discussed not having CSV as a default format. I'm going to do some experimentation locally whilst leaving it in, then we can decide before merging.